src/applications/caregivers/tests/e2e/utils/helpers.js

```diff
@@ -14,9 +14,11 @@ import mockSubmission from '../fixtures/mocks/mock-submission.json';
 export const setupPerTest = () => {
   cy.intercept('GET', '/v0/feature_toggles?*', featureToggles);
   cy.intercept('POST', 'v0/form1010cg/attachments', mockUpload);
-  cy.intercept('GET', '/v1/facilities/va?*', mockFacilities).as(
-    'getFacilities',
-  );
+  cy.intercept(
+    'GET',
+    '/v0/health_care_applications/facilities?*',
+    mockFacilities,
```

Comment:
Updated expected facilities response since we now get it from `vets-api` backend vs when it was previously directly calling the lighthouse api.